src/Factory.php

```diff
@@ -24,13 +24,15 @@
     public static function create()
     {
         // @codeCoverageIgnoreStart
-        if (function_exists('uv_default_loop')) {
+        $isPHPSeven = PHP_VERSION_ID >= 70000;
```

Comment:
Why that? `PHP_VERSION_ID >= 70000` can be optimized out by OPcache, but sure if this can. You can also use `PHP_MAJOR_VERSION >= 7`.